components/search_results/search_results.jsx

```diff
@@ -222,12 +230,28 @@ class SearchResults extends React.Component {
                     />
                 </div>
             );
+        } else if (this.props.isMentionSearch && noResults) {
+            ctls = (
+                <div
+                    className={classNames(['sidebar--right__subheader a11y__section',
+                        {'sidebar-expanded': this.props.isSideBarExpanded && noResults,
+                        }])}
+                >
+                    <NoResultsIndicator
+                        variant={NoResultsVariant.Mentions}
+                    />
+                </div>
+            );
         } else if (noResults) {
             ctls = (
-                <div className='sidebar--right__subheader a11y__section'>
-                    <NoResultSearchHint
-                        dataRetentionEnableMessageDeletion={this.props.dataRetentionEnableMessageDeletion}
-                        dataRetentionMessageRetentionDays={this.props.dataRetentionMessageRetentionDays}
+                <div
+                    className={classNames(['sidebar--right__subheader a11y__section',
+                        {'sidebar-expanded': this.props.isSideBarExpanded && noResults,
+                        }])}
+                >
+                    <NoResultsIndicator
+                        variant={NoResultsVariant.ChannelSearch}
+                        titleValues={{channelName: `"${this.props.searchTerms}"`}}
```

Comment:
`{channelName: `"${this.props.searchTerms}"`}` create a new object every time this component is rendered causing  `NoResultsIndicator ` to render as well. Here, in this case, it should be fine but just something to keep in mind.